test.sh

```diff
@@ -6,12 +6,31 @@ cd $DIR
 
 source ./setup.sh
 
+# reset safety coverage data and generate gcc note file
+rm -f ./libsafety/*.gcda
+
 # *** build ***
-scons -j8 "$@"
+scons -j8 --coverage "$@"
```

Comment:
does this need to be a flag? why not make it the default?

Reply:
don't notice any extra test time https://github.com/commaai/opendbc/pull/2327

Reply:
red diff!